src/plugins/intel_gpu/src/kernel_selector/cl_kernels/border_gpu_ref.cl

```diff
@@ -70,7 +134,11 @@ KERNEL(border_gpu_ref)(
     const uint out_w  = out_yw / OUTPUT_SIZE_Y;
 
 #ifdef BORDER_TYPE_CONSTANT
```

Comment:
Do we still need this?

Reply:
BORDER_TYPE_CONSTANT is just type of border such as BORDER_TYPE_EDGE, BORDER_TYPE_MIRROR or BORDER_TYPE_MIRROR_101